build/ci/official.yml

```diff
@@ -129,12 +129,29 @@ steps:
     ArtifactType: Container
   condition: succeededOrFailed()
 
-- task: NuGetPublisher@0
-  displayName: NuGet Publisher - Roslyn Myget
+- task: ms-vseng.MicroBuildShipTasks.7c429315-71ba-4cb3-94bb-f829c95f7915.MicroBuildCodesignVerify@2
+  displayName: Verify Signed Files
   inputs:
-    searchPattern: 'artifacts\$(BuildConfiguration)\packages\*.nupkg'
-    connectedServiceName: 'Roslyn - Myget'
-    nuGetVersion: 4.0.0.2283
+    TargetFolders: |
+      $(Build.SourcesDirectory)/artifacts/$(BuildConfiguration)/packages
+
+- task: NuGetToolInstaller@0
+  inputs:
+    versionSpec: '4.9.2'
+
+# Authenticate with service connections to be able to publish packages to external nuget feeds.
+- task: NuGetAuthenticate@0
+  inputs:
+    nuGetServiceConnections: azure-public/vs-impl
+
+# TODO - this'll most likely not work so use something like eng\publish-assets.ps1 which is used by Roslyn
+- task: NuGetCommand@2
+  displayName: NuGet push - vs-impl
+  inputs:
+    command: push
+    packagesToPush: 'artifacts\$(BuildConfiguration)\packages\*.nupkg'
```

Comment:
I know you didn't change this, but more for education, what happens with builds against private branches? How do we avoid overlap with package version numbers?

Reply:
Moving this to the release pipeline will help avoid that